editor/js/libs/ui.js

```diff
@@ -534,6 +534,27 @@ UI.Color.prototype.setHexValue = function ( hex ) {
 
 // Number
 
+UI.nudgeNumericValue = function ( keyCode, shiftKeyDown ) {
```

Comment:
This "method" is shared by two UI elements: `UI.Number` and `UI.Integer`. Below you'll find two lines that assign it as a method to each class...

```
UI.Number.prototype.nudgeNumericValue = UI.nudgeNumericValue;

...

UI.Integer.prototype.nudgeNumericValue = UI.nudgeNumericValue;
```

I'm not confident that this is the best way to do this. Suggestions welcome!